docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,162 @@
+:::
+
+## Benefits/Disadvantages of using a Dongle/Split setup
+
+Benefits:
+
+- Longer battery life for peripheral halves of the keyboard.
+- Dongle can be powered by the host device and no long wires are needed.
+- Easier to connect to the host device.
+- Only the central side needs to reflashed for keymap updates
+
+Disadvantages:
+
+- More complex setup.
+- More components needed.
+- Added latency due to the extra hop between the host device and the keyboard halves.
+- Each half of the keyboard needs its own firmware (ex: left, right, and dongle).
+
+:::warning[Reset/Bootloader behavior]
+When using a combo to trigger reset/bootloader, only the central side will be affected. To avoid this, you can avoid using the combo and instead use a physical reset button on the central half.
+That will allow you to locally reset the central/peripheral side without affecting other halves. 
+Further explanation can be found under [Behavior Reset#split-keyboards](../behaviors/reset.md#split-keyboards)
+:::
+
+## Building a Split Keyboard shield with a Dongle
+
+:::note
+Because we will be changing the roles of the keyboard halves, we will also need to flash the reset firmware to the keyboard halves.
+Information about the reset firmware can be found under Troubleshooting [Reset firmware](../troubleshooting/connection-issues.mdx#acquiring-a-reset-uf2)
+:::
+
+### Kconfig.shield
+
+First we will introduce a third split to our keyboard configuration. This will be used as the dongle.
+
+```kconfig
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD_LEFT
+    def_bool $(shields_list_contains,my_board_left)
+
+config SHIELD_MY_BOARD_RIGHT
+    def_bool $(shields_list_contains,my_board_right)
+```
+
+### Kconfig.defconfig
+
+Next we will define the roles of the keyboard halves. The left and right halves will be set as peripherals, and the dongle will be set as the central.
+We will also only give the dongle the keyboard name.
+
+```kconfig
+if SHIELD_MY_BOARD_DONGLE
+
+config ZMK_KEYBOARD_NAME
+    default "My Board"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+    default y
+
+endif
+
+if SHIELD_MY_BOARD_DONGLE || SHIELD_MY_BOARD_LEFT || SHIELD_MY_BOARD_RIGHT
+
+config ZMK_SPLIT
+    default y
+
+endif
+```
+
+### Dongle configuration
+
+Finally, we will define the dongle configuration. We want to set the number of peripherals to 2 and disable sleep mode for the dongle since it will be connected to the host device.
+In case you want to use the dongle with a battery, you can enable sleep mode for the dongle. Since the dongle will be connected to the host device, we will also increase the broadcast power.
+
+```ini title="my_board_dongle.conf"
+CONFIG_ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS=2
+
+CONFIG_ZMK_SLEEP=n
+# Uncomment if sleep is enabled
+# CONFIG_ZMK_IDLE_SLEEP_TIMEOUT=3600000
+# Make the dongle broadcast with more power
+CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
+```
```

Comment:
This stuff should be wrapped into the .defconfig. A note on waking dongles from sleep if sleep is enabled might be good.

Reply:
Moved under .defconfig